api/envoy/admin/v2alpha/config_dump.proto

```diff
@@ -285,3 +286,14 @@ message SecretsConfigDump {
   // warming in preparation to service clusters or listeners.
   repeated DynamicSecret dynamic_warming_secrets = 3;
 }
+
+// This message lists the extensions that are available in the Envoy server.
+message ExtensionsConfigDump {
```

Comment:
I'm working on a more full featured set messages for reporting envoy extension set to the xDS servers. See [spec ](https://docs.google.com/document/d/1xeVvJ6KjFBkNjVspPbY_PwEDHC7XPi0J5p1SqUXcCl8/edit#heading=h.7mzjb4t3khsf) for details.
I wonder how it would make sense to partition this work as there is significant overlap. I only have proto definitions at this point, no code. I can put up PR with just proto changes and when you can base your work on it. Or you can take the changes I already have and add them to this PR? What do you think?

Reply:
I should also mentions that it may take me a couple of day for complete PR if you can wait.

Reply:
Oh that doc is great, thanks @yanavlasov!

From the management server POV, getting the information in Node on each service is exactly what we want to ensure that we can support different features with any specific envoy server.

I think there is a case for also exposing the node information from the admin endpoints (see discussion in #9097) so that config management and other tooling can easily get it. However, after reading the doc, I'm more inclined to think we should add the `Node` message as a field in `ServerInfo`? WDYT?

Reply:
It may make sense. I think admin interface would be redesigned at some point as it is not secure as it stands and causes some headaches as a result.
In terms of implementation it should not be a problem.